comps/llms/langchain/docker/tgi/docker_compose_llm.yaml

```diff
@@ -15,7 +15,7 @@
 version: "3.8"
 
 services:
-  tgi_service:
+  vllm_service:
```

Comment:
looks incorrect image.

Reply:
Thanks for your comments. Have fixed it.